overlord/state/state.go

```diff
@@ -92,12 +92,13 @@ type State struct {
 	// for registering runtime callbacks
 	lastHandlerId int
```

Comment:
Small nitpick, I believe it would be better to move it up here and `s/noticeLastTimestamp/lastNoticeTimestamp`
```suggestion
	lastHandlerId int
	lastNoticeTimestamp time.Time
```

Reply:
Agree. Done.